### PR TITLE
integrate sql_create_date into dplyr_integration.R

### PR DESCRIPTION
**Pull request by OssiLehtinen**

### CODE REVIEW

**Comment by DyfanJones** [comment]
It is weird that codecov isn't picking up the new tests 🤔 we are calling the new methods directly with an expected result. how strange.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# noctua 1.8.1.9000
+## New Feature
+* Move `sql_escape_date` into `dplyr_integration.R` backend (#121). Thanks to @OssiLehtinen for developing Athena date translation.
```
Comment (DyfanJones) [inline NEWS.md:3]: The hyperlink to # 121 has to be set up like: `(RAthena: [# 121](https://github.com/DyfanJones/RAthena/issues/121))`. The reason for this is that github will look for issue 121 on `noctua`'s github on and not RAthena 😄

Comment (OssiLehtinen) [reply NEWS.md:3]: Ah sorry about that, did a lazy copy paste there...

**Comment by DyfanJones** [comment]
I can't see anything wrong with your code/ unit test. I will spend sometime trying to fix the code coverage after this PR. It isn't a high priority but more of a nice to have :D

**Comment by DyfanJones** [comment]
hi @OssiLehtinen, do you have anytime to update the link in the news.md? Happy to make the correction myself if your are snowed under :)

**Comment by OssiLehtinen** [comment]
Urgh, 'snowed under' sounds about right... Fixed the link now.

**Comment by DyfanJones** [comment]
Thanks @OssiLehtinen for your contribution. I will merge this PR now :D

**Pull request merged**